text/0000-is-aligned-intrinsic.md

```diff
@@ -0,0 +1,287 @@
+- Feature Name: alignto_intrinsic
+- Start Date: 2017-06-20
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Add an intrinsic (`fn alignto(ptr: *const (), align: usize) -> usize`)
+which returns the number of bytes that need to be skipped in order to correctly align the
+pointer `ptr` to `align`.
```

Comment:
bikeshed: now that this doesn't return a pointer, maybe it should have a different name?  Would help make it easier to talk about the two things, too, if the intrinsic and the eventually-stable methods had different names.